swizzle/examples/calico.yml

```diff
@@ -1,13 +1,15 @@
 ---
 all:
   vars:
+    etcd_version: v3.2.24
```

Comment:
Is there a specific reason this needs to be bumped here?

Reply:
Haha no I missed that during my diff. Rebased without it.